docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,452 @@
 
-```bash
-> docker pull vesoft/nebula-graph:nightly
-```
+In the above figure, there are two tags(**player**, **team**) and two edge types (**serve** and **follow**).
 
-## Try Nebula Graph Deployment From Docker Compose
+### Creating and Using a Graph Space
 
-The easiest way to try nebula cluster is using [docker compose](https://docs.docker.com/compose/install/). For details please refer nebula [docker deployment](https://github.com/vesoft-inc/nebula-docker-compose).
+A graph space in **Nebula Graph** is similar to a traditional database. First, you need to create a space and use it before can do any other operations.
 
-### Startup Service
+You can create and use a graph space by the following steps:
 
-When `nebula` image is ready, run a container:
+1. Enter the following statement to create a graph space:
 
-```bash
-> docker run -it vesoft/nebula-graph:nightly /bin/bash
+```ngql
+nebula> CREATE SPACE nba(partition_num=1024, replica_factor=1);
 ```
 
-After login, you're in the `root` directory and you should switch to the nebula directory
+**Note**:
 
-```bash
-> cd /usr/local/nebula/
-```
+* `partition_num` specifies the number of partitions in one replica.
+* `replica_factor` specifies the number of replicas in the cluster.
 
-Start meta service, storage service and graph service:
+2. Enter the following statement to use the graph space:
 
-```bash
-> scripts/nebula.service start all
+```ngql
+nebula> USE nba;
 ```
 
-Check services' status:
+Now you can check the space you just created by the following statement:
 
-```bash
-> scripts/nebula.service status all
+```ngql
+nebula> SHOW SPACES;
 ```
 
-### Connect to Service
+The following information is displayed:
 
-```bash
-> bin/nebula -u=user -p=password
 ```
-
-* -u is to set the user name, `user` is the default Nebula user account
-* -p is to set password, `password` is the default password for account `user`
-
-Further more, if you run the console (`bin/nebula`) in a remote host from `graphd`, please specify the server's ip and port:
-
-```bash
-> bin/nebula -u=user -p=password -addr={graphd ip} --port={graphd port}
+========
+| Name |
+========
+| nba  |
+--------
 ```
 
-If you have any questions or concerns about the deployment procedures, please do not hesitate to open an issue on [GitHub](https://github.com/vesoft-inc/nebula/issues).
-
-## Build Your Own Graph
-
-This section describes how to build a graph and run queries. The example is built on the graph below:
-
-![image](https://user-images.githubusercontent.com/42762957/68355084-24cee280-0149-11ea-8e16-79e6d4af8638.png)
-
-There are three kinds of tags (_course_, _building_ and _student_) and two edge types (_choose_ and _follow_).
+### Defining the Schema for Your Data
 
-### Create a Graph Space
+In **Nebula Graph** we classify different vertices into one group which is named a tag. The `CREATE TAG` statement defines a tag, with a type name and an attribute list. The `CREATE EDGE` statement defines an edge type.
```

Comment:
I think we should unify `attribute` to `property` since all other docs use `property`.

Reply:
Yes, both are right. People use these two alternatively. We can use attribute if you agree with me.